walkthroughs/howto-k8s-fargate/README.md

```diff
@@ -0,0 +1,53 @@
+## Prerequisites
+- Setup EKS cluster with Fargate support.
+  - You can use [clusterconfig.yaml](./clusterconfig.yaml) with [eksctl](https://eksctl.io)
+- [Walkthrough: App Mesh with EKS](../eks/)
```

Comment:
This walkthrough makes you create a cluster as well. I personally think its really confusing when walkthroughs point to other walkthroughs. Especially since this one would require you to deviate from the instructions in the pre-requisite walkthrough.

Reply:
Fair enough, there is no need to follow the EKS walkthrough. Cluster created under this walkthrough should work as-is.